content/en/docs/setup/production-environment/index.md

```diff
@@ -0,0 +1,256 @@
+---
+title: "Production environment"
+description: Create a production-quality Kubernetes cluster
+weight: 30
+no_list: true
+---
+<!-- overview -->
+
+A production-quality Kubernetes cluster requires planning and preparation.
+If your Kubernetes cluster is to run critical workloads, it must be configured to be resilient.
+
```

Comment:
```suggestion
This page explains steps you can take to set up a production-ready cluster,
or to uprate an existing cluster for production use.

If you're already familiar with production setup and want the links, skip to
[What's next](#what-s-next).

```
_and_ add a What's Next section at the end of the page with a summary of links.
At a minimum I would include:
- each of the tools listed in https://kubernetes.io/docs/setup/production-environment/tools/
- a link to https://kubernetes.io/docs/setup/production-environment/turnkey-solutions/

and possibly:
- https://kubernetes.io/docs/setup/best-practices/certificates/#configure-certificates-manually
```

Reply:
Done.